util/regexp-assemble/data/942190.data

```diff
@@ -45,3 +45,23 @@
 \bunion select @
 \bunion[\w(\s]*?select
 \buser\s*?\([^\)]*?
+
+##! Added (?:\/\*.*?\*\/)?\s* to support comment block that can be used to bypass the filter
+##! Replaced lazy \s? to greedy \s* and whitespace ' ' with \s
+
+[\"'`]\s*(?:\/\*.*?\*\/)?\s*!\s*(?:\/\*.*?\*\/)?\s*[\"'`\w]
+[\"'`];?\s*(?:\/\*.*?\*\/)?\s*having\b\s*(?:\/\*.*?\*\/)?\s*[^\s]
+[\"'`];?\s*(?:\/\*.*?\*\/)?\s*select\b\s*(?:\/\*.*?\*\/)?\s*[^\s]
+[\"'`];?\s*(?:\/\*.*?\*\/)?\s*union\b\s*(?:\/\*.*?\*\/)?\s*all\b\s*(?:\/\*.*?\*\/)?\s*[^\s]
+[\"'`];?\s*(?:\/\*.*?\*\/)?\s*union\b\s*(?:\/\*.*?\*\/)?\s*distinct\b\s*(?:\/\*.*?\*\/)?\s*[^\s]
+[\"'`];?\s*(?:\/\*.*?\*\/)?\s*union\b\s*(?:\/\*.*?\*\/)?\s*select\b\s*(?:\/\*.*?\*\/)?\s*[^\s]
+\s*(?:\/\*.*?\*\/)?\s*exec.*?\Wxp_cmdshell
+\s*(?:\/\*.*?\*\/)?\s*execute.*?\Wxp_cmdshell
+\Wiif\s*(?:\/\*.*?\*\/)?\s*\(
+\bconnection_id\s*(?:\/\*.*?\*\/)?\s*\([^\)]*?
+\bcurrent_user\s*(?:\/\*.*?\*\/)?\s*\([^\)]*?
+\bdatabase\s*(?:\/\*.*?\*\/)?\s*\([^\)]*?
+\binto[\s+]+dumpfile\s*(?:\/\*.*?\*\/)?\s*[\"'`]
+\binto[\s+]+outfile\s*(?:\/\*.*?\*\/)?\s*[\"'`]
+\bschema\s*(?:\/\*.*?\*\/)?\s*\([^\)]*?
+\buser\s*(?:\/\*.*?\*\/)?\s*\([^\)]*?
```

Comment:
Why didn't you update
- `\bunion select @`
- `\bexec\s+master\.`
- `\bexecute\s+master\.`
?